google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/BigQueryReadClientTest.java

```diff
@@ -0,0 +1,165 @@
+            .setTransportChannelProvider(channelProvider)
+            .setCredentialsProvider(NoCredentialsProvider.create())
+            .build();
+    client = BigQueryReadClient.create(settings);
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    client.close();
+  }
+
+  @Test
+  @SuppressWarnings("all")
+  public void createReadSessionTest() {
+    String name = "name3373707";
+    String table = "table110115790";
+    ReadSession expectedResponse = ReadSession.newBuilder().setName(name).setTable(table).build();
+    mockBigQueryRead.addResponse(expectedResponse);
+
+    String parent = "parent-995424086";
+    ReadSession readSession = ReadSession.newBuilder().build();
+    int maxStreamCount = 940837515;
+
+    ReadSession actualResponse = client.createReadSession(parent, readSession, maxStreamCount);
+    Assert.assertEquals(expectedResponse, actualResponse);
+
+    List<AbstractMessage> actualRequests = mockBigQueryRead.getRequests();
+    Assert.assertEquals(1, actualRequests.size());
+    CreateReadSessionRequest actualRequest = (CreateReadSessionRequest) actualRequests.get(0);
+
+    Assert.assertEquals(parent, actualRequest.getParent());
+    Assert.assertEquals(readSession, actualRequest.getReadSession());
+    Assert.assertEquals(maxStreamCount, actualRequest.getMaxStreamCount());
+    Assert.assertTrue(
+        channelProvider.isHeaderSent(
+            ApiClientHeaderProvider.getDefaultApiClientHeaderKey(),
+            GaxGrpcProperties.getDefaultApiClientHeaderPattern()));
+  }
+
+  @Test
+  @SuppressWarnings("all")
+  public void createReadSessionExceptionTest() throws Exception {
+    StatusRuntimeException exception = new StatusRuntimeException(Status.INVALID_ARGUMENT);
+    mockBigQueryRead.addException(exception);
+
+    try {
+      String parent = "parent-995424086";
+      ReadSession readSession = ReadSession.newBuilder().build();
+      int maxStreamCount = 940837515;
+
+      client.createReadSession(parent, readSession, maxStreamCount);
+      Assert.fail("No exception raised");
+    } catch (InvalidArgumentException e) {
+      // Expected exception
+    }
+  }
+
+  @Test
+  @SuppressWarnings("all")
+  public void readRowsTest() throws Exception {
+    long rowCount = 1340416618L;
+    ReadRowsResponse expectedResponse = ReadRowsResponse.newBuilder().setRowCount(rowCount).build();
+    mockBigQueryRead.addResponse(expectedResponse);
+    ReadRowsRequest request = ReadRowsRequest.newBuilder().build();
+
+    MockStreamObserver<ReadRowsResponse> responseObserver = new MockStreamObserver<>();
+
+    ServerStreamingCallable<ReadRowsRequest, ReadRowsResponse> callable = client.readRowsCallable();
+    callable.serverStreamingCall(request, responseObserver);
+
+    List<ReadRowsResponse> actualResponses = responseObserver.future().get();
+    Assert.assertEquals(1, actualResponses.size());
+    Assert.assertEquals(expectedResponse, actualResponses.get(0));
+  }
+
+  @Test
+  @SuppressWarnings("all")
```

Comment:
what warnings are being surpressed?

Reply:
Warning that is suppressed:
```
Warning:(147, 13) java: serverStreamingCall(RequestT,com.google.api.gax.rpc.ApiStreamObserver<ResponseT>) in com.google.api.gax.rpc.ServerStreamingCallable has been deprecated
```
This also what the auto-generated unit test BaseBigQueryReadClientTest.java is using as a suppression.